packages/core/src/commands/splitListItem.ts

```diff
@@ -19,7 +19,7 @@ declare module '@tiptap/core' {
   }
 }
 
-export const splitListItem: RawCommands['splitListItem'] = typeOrName => ({
+export const splitListItem: RawCommands['splitListItem'] = (typeOrName, overrideAttrs = {}) => ({
```

Comment:
Maybe I'm a bit blind but won't Typescript define the type for `overrideAttrs` to an empty object? Don't you need to add a type for `overrideAttrs` here?

https://github.com/ueberdosis/tiptap/blob/fc62eb3c4d6d784b5f9b43a58e0bb0a1073cb612/packages/core/src/commands/splitListItem.ts#L17C23-L17C23

I'd suggest just to set it to `[key: string]: any`?

Reply:
Good catch!